### PR TITLE
Add Busy beta opt-in

### LINKED ISSUE #1253: Add setting to be beta tester
We can add a setting to enable user to beta test the last version of busy (staging.busy.org). If user active it, he will get redirected to staging.busy.org automatically if he try to go on busy.org (until he disable it).

### PR DESCRIPTION
**Pull request by Sekhmet**
Fixes #1253 

Changes:
- Add beta checkbox.
- Redirect client-side.
- Redirect server-side.

### CODE REVIEW

**Comment by bonustrack** [comment]
I have a small issue, when i open the page setting from a fresh tab, i will see the page loaded for few milisecond then it disappear and loading appear. The setting page should never be visible before been loaded. See: 

bug

**Comment by Sekhmet** [comment]
This bug is not related to that PR (it happens on busy.org as well).

File: src/client/settings/settingsReducer.js [hunk 1]
```diff
@@ -53,6 +56,7 @@ const settings = (state = initialState, action) => {
         showNSFWPosts: action.payload.showNSFWPosts,
         upvoteSetting: action.payload.upvoteSetting,
         rewardSetting: action.payload.rewardSetting,
+        useBeta: !!action.payload.useBeta,
```
Comment (jm90m) [inline src/client/settings/settingsReducer.js]: isn't `useBeta` always a boolean? so you won't need the `!!`

File: src/client/settings/__tests__/settingsReducer.js [hunk 2]
```diff
@@ -85,7 +86,10 @@ describe('settingsReducer', () => {
       },
     };
 
-    expect(settingsReducer(stateBefore, action)).to.eql(stateAfter);
+    const res = settingsReducer(stateBefore, action);
+    console.log(res);
```
Comment (jm90m) [inline src/client/settings/__tests__/settingsReducer.js]: random `console.log`?

File: src/client/settings/settingsReducer.js (hunk 1 above)
Comment (Sekhmet) [reply src/client/settings/settingsReducer.js]: It comes from SC2 so it might be `undefined` as well. This way it won't safe as `undefined` in state.

Comment (jm90m) [reply src/client/settings/settingsReducer.js]: ahh I see, makes sense

**Pull request merged**